code/espurna/config/progmem.h

```diff
@@ -208,7 +211,9 @@ PROGMEM const unsigned char magnitude_decimals[] = {
     3, 0, 0, 0, 0, 0, 0, 0, // Power decimals
     0, 0, 0, // analog, digital, event
     0, 0, 0, // PM
-    0, 0, 3, 3, 0,
+    0, 0,
+    0, 0, 3, // UVA, UVB, UV Index
```

Comment:
@xoseperez UVA, UVB and UVI as returned as floats. Should we keep them with decimals when presenting?

Reply:
Even if they are returned as floats, the decimals value must be that that makes more sense. If UVA and UVB are integers then set it to 0.